licenses/LICENSE-copybutton.txt

```diff
@@ -0,0 +1,254 @@
+A. HISTORY OF THE SOFTWARE
```

Comment:
this looks like license for python?

Reply:
Yes, this is the license for python. Do you want me to just put `PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2` under B in the licenses file?

Reply:
I'm OK with trimming it down to the 'section B' part as I agree that the rest is not licensing terms. You can modify the heapq license file too accordingly.